ocaml/utils/language_extension.mli

```diff
@@ -103,11 +103,27 @@ val with_disabled : 'a t -> (unit -> unit) -> unit
     way. *)
 val restrict_to_erasable_extensions : unit -> unit
 
-(** Permanently ban all extensions; used for [-disable-all-extensions] to
-    ensure that some code is 100% extension-free.  When called, disables any
+(** Permanently ban all extensions; used for [-disable-all-extensions] to ensure
+    that some code is 100% extension-free.  When called, disables any
     currently-enabled extensions, including the defaults.  Causes any future
-    uses of [set ~enabled:true], [enable], and their [with_] variants to
-    raise; also causes any future uses of [only_erasable_extensions] to raise.
+    uses of [set ~enabled:true], [enable], and their [with_] variants to raise;
+    also causes any future uses of [restrict_to_erasable_extensions] to raise.
     The [is_enabled] function will still work, it will just always return
     [false].*)
 val disallow_extensions : unit -> unit
+
+(**/**)
+
+(** Special functionality that can only be used in "pprintast.ml" *)
+module For_pprintast : sig
+  (** A fully-polymorphic function that can wrap printers from "pprintast.ml" *)
```

Comment:
I might consolidate this comment with the meat of the one on `make_printer_exporter` (possibly reducing the latter's comment to "Raises if called more than once ever"). It's obvious that the thing named `make_X` (with unit argument) makes an `X`.

I might reword this one to something like:

```
(** A function for wrapping a printer from "pprintast.ml" so that it will unconditionally print
   Jane Syntax instead of raising an exception on disabled extensions.
*)
```
It's fully polymorphic for an uninteresting reason so it's a bit odd to highlight that in the comment. If you think it's worth mentioning this at all, I think the comment can be reduced to an "N.B.", like "N.B. we universally quantify the ['a] type variable so we don't run afoul of the value restriction in [make_printer_exporter]"